packages/components/src/form-token-field/index.tsx

```diff
@@ -0,0 +1,686 @@
+/**
+ * External dependencies
+ */
+import {
+	last,
+	take,
+	clone,
+	uniq,
+	map,
+	difference,
+	each,
+	identity,
+	some,
+} from 'lodash';
+import classnames from 'classnames';
+import type { KeyboardEvent, MouseEvent, TouchEvent } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useRef, useState } from '@wordpress/element';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { useDebounce, useInstanceId, usePrevious } from '@wordpress/compose';
+import { speak } from '@wordpress/a11y';
+import {
+	BACKSPACE,
+	ENTER,
+	UP,
+	DOWN,
+	LEFT,
+	RIGHT,
+	SPACE,
+	DELETE,
+	ESCAPE,
+} from '@wordpress/keycodes';
+import isShallowEqual from '@wordpress/is-shallow-equal';
+
+/**
+ * Internal dependencies
+ */
+import Token from './token';
+import TokenInput from './token-input';
+import SuggestionsList from './suggestions-list';
+import type { FormTokenFieldProps, TokenItem } from './types';
+
+export function FormTokenField( props: FormTokenFieldProps ) {
```

Comment:
We also want to copy the main component description and example from the README and add it as a JSDoc comment here ([example](https://github.com/WordPress/gutenberg/blob/943cec92f21fedcd256502ea72d9903941f3b05a/packages/components/src/unit-control/index.tsx#L290-L306)). Similar to the inline descriptions for the prop types, it'll become accessible via IntelliSense and the Storybook docs.